src/plugins/canvasAnnotationsPlugin.js

```diff
@@ -26,12 +46,21 @@ class CanvasAnnotationsWrapper extends Component {
           config,
           receiveAnnotation,
           storageAdapter: config.annotation.adapter,
+          toggleSingleCanvasDialogOpen: this.toggleSingleCanvasDialogOpen,
           windowId: targetProps.windowId,
+          windowViewType,
         }}
       >
         <TargetComponent
           {...props} // eslint-disable-line react/jsx-props-no-spreading
         />
+        {windowViewType !== 'single' && (
```

Comment:
Can you say more about why we need to render the component here as well? I'm not sure of the case when this should render. Thanks!

Reply:
Ah.. nevermind. ++ @camillevilla for pointing me to the editing view.